editor/editor_properties.cpp

```diff
@@ -3175,7 +3175,18 @@ bool EditorPropertyNodePath::is_drop_valid(const Dictionary &p_drag_data) const
 		return false;
 	}
 	Array nodes = p_drag_data["nodes"];
-	return nodes.size() == 1;
+	if (nodes.size() != 1) {
+		return false;
+	}
+
+	Node *dropped_node = get_tree()->get_edited_scene_root()->get_node(nodes[0]);
```

Comment:
You could add `ERR_FAIL_NULL` here in case the path is invalid for whatever reason.

Reply:
Done! Thanks for the guidance, I'm new to this :)